saleor/core/utils/url.py

```diff
@@ -15,10 +17,14 @@ def validate_storefront_url(url):
         parsed_url = urlparse(url)
         domain, _ = split_domain_port(parsed_url.netloc)
     except ValueError as error:
-        raise ValidationError({"redirectUrl": str(error)})
+        raise ValidationError(
+            {"redirectUrl": str(error)}, code=AccountErrorCode.INVALID
```

Comment:
Same as above. We shouldn't import any saleor's modules in core. This is a perfect wat to get more circular imports.
If we need to import `AccountErrorCode` it means that this class or `validate_storefront_url` is declared in wrong place.

Reply:
It will stay as it is for now; I've created a ticket for that: #4744